HTML/Cookie Clicker.html

```diff
@@ -1,5 +1,7 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
+<!-- saved from url=(0049)https://ktane.timwi.de/HTML/Cookie%20Clicker.html -->
 <html>
+<link type="text/css" id="dark-mode" rel="stylesheet" href="https://ktane.timwi.de/HTML/Cookie%20Clicker.html">
```

Comment:
This line should probably be removed or if it has any importance then it should be in `<head>`

Reply:
This line is indeed quite invalid, as it says it links to a CSS style sheet but the link is actually to an HTML page